embedchain/apps/PersonApp.py

```diff
@@ -22,6 +22,23 @@ def __init__(self, person, config: BaseAppConfig = None):
         self.person_prompt = f"You are {person}. Whatever you say, you will always say in {person} style."  # noqa:E501
         super().__init__(config)
```

Comment:
shouldn't this throw an error? What super is it calling, it does not have a parent class.

Reply:
if I am not wrong this class is being used for multiple inheritance and hence it might be required